src/docfx/docfx.csproj

```diff
@@ -36,7 +36,7 @@
     <PackageReference Include="CommandLineParser" Version="1.9.71" />
     <PackageReference Include="NuGet.Common" Version="5.7.0" />
     <PackageReference Include="Owin" Version="1.0.0" />
-    <PackageReference Include="Microsoft.Owin" Version="4.1.1" />
+    <PackageReference Include="Microsoft.Owin" Version="4.2.2" />
     <PackageReference Include="Microsoft.Owin.FileSystems" Version="4.1.1" />
```

Comment:
Do these dependencies need to upgrade to 4.2.2 too?

Reply:
Sorry， didn't notice it, then we just need to release the change to main branch，there is a task in the sprint to handle the release, @YunshuMeg you can close the task and PR directly.